### PR TITLE
Refactor: landing, CTA, footer sections

### PR DESCRIPTION
**Pull request by cp-shruti-s**
**Tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [x] Laptop / Macbook  (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

Screenshot from 2023-01-02 17-44-43

### CODE REVIEW

**Comment by cp-sumi-k** [comment]
Some changes based on figma design
- Use opacity on color instead of direct color
- Gradient lines should start from pink color
- Increase vertical padding of button
- Size of `bottom fonts` and `40+ reviews` is larger then figma
- Background image is not properly rendered in all devices
- Main image is smaller then figma image

File: vue-frontend/src/components/home-new/LandingSection.vue [hunk 1]
```diff
@@ -143,22 +151,22 @@ export default {
         {
           id: 1,
           className: "tw-z-0",
-          image: review1,
+          image: review4,
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/LandingSection.vue:154]: I think it should be `client` instead of `review`

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/partials/NewHeader.vue [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      lastScrollY: 0,
+      currentRoutePath: this.$router.currentRoute._value.path,
+      navbars: [
+        {
+          name: "Home",
+          url: "/",
+          className:
+            "v2-normal-3-text tw-text-[#000000de] tw-font-Inter tw-tracking-[0] tw-mt-0 tw-mb-0 tw-ml-0 tw-p-0 tw-mr-[20px] sm:tw-mr-[30px] tw-inline-block tw-relative after:tw-absolute after:tw-w-full after:tw-scale-x-0 after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-pink-300 after:tw-origin-bottom-left after:tw-duration-300 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left",
+          target: false,
+          event: this.isJobsUrl ? "tap_jobs_header_home" : " ",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/partials/NewHeader.vue:108]: Let's remove analytics events for new design for now

File: vue-frontend/src/components/home-new/LandingSection.vue [hunk 3]
```diff
@@ -1,164 +1,177 @@
 <template>
-  <section>
+  <div class="tw-relative tw-px-0 lg:tw-py-[50px] tw-overflow-hidden">
+    <img
+      :src="bg2400"
+      :srcset="`${bg400} 400w, ${bg800} 800w, ${bg1200} 1200w, ${bg1200} 2400w`"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/LandingSection.vue:5]: `${bg1200} 2400w `?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/LandingSection.vue (hunk 3 above)
Comment (cp-shruti-s) [reply vue-frontend/src/components/home-new/LandingSection.vue:5]: I was checking that 1200 width image is set on 2400 width screen

Comment (cp-sumi-k) [reply vue-frontend/src/components/home-new/LandingSection.vue]: It should 2400. How 1200 can fit in 2400? It will definitely cut from borders

Comment (cp-shruti-s) [reply vue-frontend/src/components/home-new/LandingSection.vue]: But 2400 width image does not fit on 2400 width screen

File: vue-frontend/src/components/home-new/LandingSection.vue [hunk 4]
```diff
@@ -1,176 +1,173 @@
...
+          class="tw-w-[100%] sm:tw-w-[85%] tw-pb-5 lg:tw-pb-8 xl:tw-pb-10 v2-header-3-text tw-tracking-[0] tw-text-[1.875rem] tw-leading-[2.5rem] md:tw-text-[2.65625rem] md:tw-leading-[3.28125rem] lg:tw-text-[3.28125rem] lg:tw-leading-[3.70625rem] 2xl:tw-text-[4.0625rem] 2xl:tw-leading-[4.6875rem] tw-text-black-core/[.87] tw-font-roboto-bold"
+        >
+          For Over A Decade We Have Helped Businesses Grow Exponentially.
+          <div>
+            — <span class="canopas-gradient-text"> Is Yours Next?</span>
+          </div>
+        </div>
         <div
-          class="tw-container tw-flex tw-flex-col tw-items-center lg:tw-flex-row-reverse lg:tw-items-start"
+          class="tw-w-[85%] md:tw-w-[65%] tw-pb-[29.5px] md:tw-pb-[47px] v2-normal-text tw-tracking-[0] tw-text-[1rem] tw-leading-[1.375rem] md:tw-text-[1.09375rem] md:tw-leading-[1.53125rem] lg:tw-text-[1.1875rem] lg:tw-leading-[1.75rem] tw-text-black-core/[0.6] !tw-font-sfpro-regular"
```
Comment (cp-priyanka-g) [inline vue-frontend/src/components/home-new/LandingSection.vue:33]: You should use either v2-normal-text or  tw-font-sfpro-regular.

**Review by cp-priyanka-g — DISMISSED** [review]

File: vue-frontend/src/components/partials/NewHeader.vue [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+              :key="navbar"
+              class="tw-my-2 sm:tw-my-0 tw-ml-0 tw-p-0"
+            >
+              <router-link
+                v-if="!navbar.target"
+                :to="navbar.url"
+                :class="[
+                  navbar.className
+                    ? navbar.className
+                    : 'tw-inline-block tw-relative tw-my-0 tw-ml-0 tw-mr-[20px] sm:tw-mr-[30px] tw-p-0 after:tw-absolute after:tw-w-full after:tw-h-[3px] after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-pink-300 after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-text-transparent hover:tw-from-[#ff9472] hover:tw-to-[#f2709c]',
```
Comment (cp-priyanka-g) [inline vue-frontend/src/components/partials/NewHeader.vue:47]: follow the Guideline of ordering classes.

File: vue-frontend/src/components/home-new/CTASection.vue [hunk 6]
```diff
@@ -1,58 +1,62 @@
...
+  >
     <img
       :src="bg2400"
       :srcset="`${bg400} 400w, ${bg800} 800w, ${bg1200} 1200w, ${bg2400} 2400w`"
       class="tw-absolute tw-top-0 tw-left-0 tw-w-full tw--z-[1]"
       alt="canopas-cta"
     />
     <div
-      class="tw-container sm:tw-w-4/5 lg:tw-w-[70%] tw-m-auto tw-px-[25px] tw-pt-[20%] md:tw-pt-[10%] tw-pb-[5%] tw-text-center tw-opacity-1"
+      class="tw-container tw-m-auto md:tw-w-[86%] lg:tw-w-4/5 xl:tw-w-[86%] tw-px-[25px] tw-pt-[20%] md:tw-pt-[10%] tw-pb-[5%] tw-opacity-1 tw-text-center"
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/CTASection.vue:12]: We should use `PX` instead of the `percentage` for the top bottom padding

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**